Parser/asdl_c.py

```diff
@@ -1093,12 +1094,7 @@ def visitModule(self, mod):
         self.emit("static int", 0)
         self.emit("astmodule_exec(PyObject *m)", 0)
         self.emit("{", 0)
-        self.emit('astmodulestate *state = get_ast_state(m);', 1)
-        self.emit("", 0)
-
-        self.emit("if (!init_types(state)) {", 1)
-        self.emit("return -1;", 2)
-        self.emit("}", 1)
+        self.emit('struct ast_state *state = get_ast_state();', 1)
```

Comment:
I guess we need to check for `state == NULL` here (in case of init_types failed)

Reply:
Oh right, fixed, thanks.